.github/workflows/python-app.yml

```
@@ -165,13 +165,55 @@ jobs:
           name: FontraPakWindows
           path: ./dist/*.exe
 
+  build-linux-binary:
+    runs-on: ubuntu-22.04
+
+    permissions:
+      contents: read
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v5
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v5
+        with:
+          node-version: 20.x
+          cache: "npm"
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements.txt
+          python -m pip install -r requirements-dev.txt
+
+      - name: Build exe
+        run: |
+          pyinstaller FontraPak.spec -y
+          mv dist/Fontra\ Pak dist/fontrapak
+      - name: Run tests
+        run: |
+          pytest
+
+      - name: Storing Ubuntu Artifacts
+        id: upload_artifact
+        uses: actions/upload-artifact@v5
+        with:
+          name: fontrapak
```

Can we use the same naming/styling convention as the other two builds?
```suggestion
          name: FontraPakUbuntu
```
Or `FontraPakLinux`?